ee/tabby-webserver/src/schema/mod.rs

```diff
@@ -434,10 +435,20 @@ impl Mutation {
         smtp_username: String,
         smtp_password: Option<String>,
         smtp_server: String,
+        from_address: Option<String>,
+        encryption: Encryption,
+        auth_method: AuthMethod,
     ) -> Result<bool> {
```

Comment:
require admin.

also for `email_setting` query and `delete_email_setting` mutation.